README.md

```diff
@@ -1,2 +1,18 @@
 # my_second_repo_2020
 My 5th lab.
+ 
+## Опис завдання:
```

Comment:
I'm form London, the capital of Great Britain, so be kind to replace this symbols in READMY.md with understandable english letters